jellyfin_kodi/full_sync.py

```diff
@@ -223,7 +223,12 @@ def process_library(self, library_id):
                 if not sync_id or sync_id == 'Refresh':
                     libraries = self.get_libraries()
                 else:
-                    libraries = self.get_libraries(sync_id)
+                    res = self.get_library(sync_id)
+                    if res is not None:
+                        # FIXME: This is a hack. Fix plz.
+                        libraries = [(sync_id,), res]
```

Comment:
I feel like it makes more sense to change the sql query so that `get_libraries()` and `get_library()` both return a consistent format instead of doing this

Reply:
Also, this is the wrong format and still fails.  `get_libraries()` returns the format
```
[(u'a656b907eb3a73532e40e44b968d0225', u'Shows', u'tvshows'),
 (u'f137a2dd21bbc1b99aa5c0f6bf02a805', u'Movies', u'movies'),
 (u'9d7ad6afe9afa2dab1a2f6e00ad28fa6', u'Collections', u'boxsets'),
 (u'7e64e319657a9516ec78490da03edccb', u'Music', u'music'),
 (u'1071671e7bffa0532e930debee501d2e', u'Playlists', u'playlists'),
 (u'89c5dc02e07352fb47c897776e0378fa', u'Playlists', u'playlists')]
 ```
While this is outputting
```
[(u'f137a2dd21bbc1b99aa5c0f6bf02a805',), (u'Movies', u'movies')]
```

Reply:
That was supposed to be a `+`, not `,`.

And I agree, but I didn't feel like carefully redoing all the places where the two item return is expected.
This is where a named tuple would be helpful.